pymatgen/analysis/phase_diagram.py

```diff
@@ -450,9 +481,15 @@ def stable_entries(self):
         """
         return self._stable_entries
 
-    def get_stable_entries_normed(self, mode="formula_unit"):
+    @lru_cache(1)  # cache in case of repeated calls
+    def get_stable_entries_normed(self, mode="atom"):
```

Comment:
In the interests of backwards compatibility (or, principle of least surprise for other users), why is the default changed here?

Reply:
Default probably doesn't need to change as I have changed the kwarg in `get_quasi_e_to_hull`. Using `atom` rather than `formula_unit` was needed there to resolve an edge case where a copy of the entry that was normalised according  to `atom` first didn't give the same energy to hull as the unnormalised copy.

I will change the default back here